src/cmd-buildextend-installer

```diff
@@ -168,8 +167,6 @@ def generate_iso():
     if is_fulliso:
         tmp_initramfs = os.path.join(tmpdir, 'initramfs')
 
-        if img_metal is None:
```

Comment:
If it helps BTW I am fine deleting the `fulliso` codepath as part of this.  Osmet obsoletes it from my PoV.

Reply:
I would much love to remove codepaths that we can. If osmet obsoletes it that's even better because we've solved the fulliso use case with the liveiso codepath.

Reply:
It's not a big deal to support both code-wise, but I do agree it's obsolete, so will remove it entirely.